src/Compilers/CSharp/Portable/DocumentationComments/DocumentationCommentIDVisitor.PartVisitor.cs

```diff
@@ -270,6 +270,20 @@ public override object VisitDynamicType(DynamicTypeSymbol symbol, StringBuilder
                 return null;
             }
 
+            private object VisitTupleType(TupleTypeSymbol symbol, StringBuilder builder)
```

Comment:
>VisitTupleType [](start = 27, length = 14)

Discussed the problem with Chris, because I didn't quite understand the logic below. He's going to try another approach.